### PR TITLE
feat(PageHero): don't hide video on xs screens

### PR DESCRIPTION
**Pull request by christian-dinh**
### Overview

as a parting gift to us, Dev redesigned the video hero to address accessibility concerns. The video is no longer hidden on xs screens and instead just wraps to the next line

I also refactored slightly so that column sizes for the left and right hero columns are defined in the same file

### PR Checklist

- [x] [Related to designs](https://www.figma.com/file/EDtIzUHehl8njnjPStWtYR/Careers-Page---Refresh---1.19.21?node-id=139%3A736)
- [x] Related to JIRA ticket: [REACH-623](https://codecademy.atlassian.net/browse/REACH-623)
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

**Review by JoshuaKGoldberg — APPROVED** [review]
awww Dev!

File: packages/gamut-labs/src/landingPage/PageHeroMedia.tsx [hunk 1]
```diff
@@ -1,45 +1,46 @@
...
-const StyledVideo = styled(Video)`
-  align-self: center;
+const RightColumn = styled(Column)<{ hideOnMobile?: boolean }>`
+  align-items: center;
+  margin-left: 1rem;
+
+  @media only screen and (max-width: ${breakpoints.sm}) {
+    ${({ hideOnMobile }) =>
+      hideOnMobile ? 'display: none;' : 'margin-left: 0;'}
+  }
```
Comment (rogervera) [inline packages/gamut-labs/src/landingPage/PageHeroMedia.tsx:20]: Can we flip the logic here to be mobile-first or is using `max-width` a necessity?

**Review by rogervera — APPROVED** [review]
For DEV!

File: packages/gamut-labs/src/landingPage/PageHeroMedia.tsx (hunk 1 above)
Comment (christian-dinh) [reply packages/gamut-labs/src/landingPage/PageHeroMedia.tsx:20]: good point I'll flip it

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.2.17-alpha.979063.0<br/>@codecademy/gamut-labs@11.4.0-alpha.979063.0<br/>@codecademy/styleguide@30.7.1-alpha.979063.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://604fdb7a89994b171534e18a--gamut-preview.netlify.app>https://604fdb7a89994b171534e18a--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/604fdb7a89994b171534e18a>Deploy Logs</a></p>

**Pull request merged**